deps/process.lua

```diff
@@ -56,18 +56,19 @@ local lenv = {}
 function lenv.get(key)
   return lenv[key]
 end
-setmetatable(lenv, {
-  __pairs = function(table)
-    local keys = env.keys()
-    local index = 0
-    return function(...)
-      index = index + 1
-      local name = keys[index]
-      if name then
-        return name, table[name]
-      end
+function lenv.iterate()
+  local keys = env.keys()
+  local index = 0
+  return function(...)
+    index = index + 1
+    local name = keys[index]
+    if name then
+      return name, env.get(name)
     end
-  end,
+  end, keys, nil
+end
+
+setmetatable(lenv, {
```

Comment:
Should still set `__pairs` to `lenv.iterate` for backwards compatibility

Reply:
Maybe luajit compile without COMPAT to lua 5.2, so change this, and now make __pairs to lenv.iterate.